Checking the Lua version inside the headers requires compiling a small
C program. However, autoconf will raise an error when attempting to do so
while cross-compiling. Hence, this specific test is removed.

Signed-off-by: Assaf Inbal <shmuelzon@gmail.com>
--- a/configure.ac	2013-06-20 07:23:27.943091833 +0300
+++ b/configure.ac	2013-06-20 07:23:30.644441710 +0300
@@ -105,7 +105,7 @@
 
 dnl Lua 5.1 or 5.2
 AX_PROG_LUA(501, 503)
 AX_LUA_HEADERS
-AX_LUA_HEADERS_VERSION(501, 503)
+#AX_LUA_HEADERS_VERSION(501, 503)
 AC_SUBST(LUA)
 AC_SUBST(LUA_INCLUDE)
